getting-started/project-setup.adoc

```diff
@@ -105,36 +105,37 @@ You will get some Maven stream once again, and at the end of it you should have:
 == Editing within VS Code
 
 . Open the project in VS Code.
-  From here we can see the workspace's contents and if we expand the contents of `/src/main` you will see the creation of several artifacts. 
+  From here we can see the workspace's contents, and if we expand the contents of `/src/main` you will see the creation of several artifacts. 
   Within `java` you will have a `GreetingResource.java` and within `resources` you will have an `application.properties` and `pricing.dmn` file. 
   These are sample files that can be later modified or deleted.
 +
 image::workspace-layout.png[VSCode Workspace Layout]
 
 . Install {VS_CODE_BUNDLE_LINK}[*{VS_CODE_BUNDLE}*].
   This gives you access to the visual editors for DMN and BPMN files.
-  It also allows you to create and edit testing scenarios for project.
+  It also allows you to create and edit testing scenarios for projects.
 +
 image::bamoe-vscode-marketplace.png[{VS_CODE_BUNDLE} within the marketplace]
 
 . Let's first click the `pricing.dmn` file to open it. 
-  When you do so you may be greeted with a message similar to `This diagram does not have layout information. Click 'Yes' to compute optimal layout, it takes time according to the diagram size. 
+  When you do so, you may be greeted with a message similar to `This diagram does not have layout information. Click 'Yes' to compute optimal layout, it takes time according to the diagram size. 
   Click 'No' to proceed without layout. 
-  Please save the layout changes once diagram is opened.` - if so click `Yes` to automap the DMN locations.
+  Please save the layout changes once diagram is opened.` 
+  If so click `Yes` to automap the DMN locations.
```

Comment:
Here I brainstorm how we could make documentation shorter. What is your opinion, if we would not repeat the text from the screenshot. Maybe something like:

If you are prompted with the dialogue below, click Yes to automap DMN locations or No to proceed without layout.

Reply:
We should be describing it in the text for accessibility, or it needs to be an alt for the image.

Reply:
I haven't seen us using that long alt attributes elsewhere, so I think I'll keep it as is for accessibility consistency if all agree

Reply:
ok